### ISSUE TITLE
Question about Robothor x_display gpu id

### ISSUE THREAD

**Issue by MarSaKi** [real-stanford/cow#2]
Hi, Samir,

Thanks for your great work! 
I have some questions about the x_display id of Robothor, could you please give me some hints?
Specifically, I have an 8 GPU machine and I want to run CoW on 4 GPUs. (e.g., CUDA_VISIBLE_DEVICES=4,5,6,7).
I modified your [code](https://github.com/columbia-ai-robotics/cow/blob/901cf3377da4297b3b4b9fc3cdde8f254f8f2849/robothor_challenge.py#L438) like this:
```
num_devices = torch.cuda.device_count()
processes = []
for worker_ind in range(nprocesses):
    class_remap = None
    if self.class_remap is not None:
        class_remap = deepcopy(self.class_remap)
    thread_controller_kwargs = deepcopy(self.controller_kwargs)
    thread_controller_kwargs['gpu_device'] = worker_ind % num_devices
    thread_controller_kwargs['x_display'] = f':0.{worker_ind % num_devices}'
```
By doing this, the model is running on gpu 4,5,6,7. But the Robothor Simulator still runs on gpu 0,1,2,3. 
In other words, 'x_display' seems to correspond to the physical GPU device ids. Do you have any idea to fix this problem?
Thanks for your attention to this matter!

**Comment by sagadre** [comment]
hi! thanks for the interest in the work! a quick hack may be to make sure `x_display` is set to `0.4`, `0.5`, ... for example by adding 4 to the `worker_idx`

for a more permanent solution, you may want to look at this issue: https://github.com/allenai/ai2thor/issues/1006

**Comment by MarSaKi** [comment]
Thanks for your quick reply!
